src/chain_spec.rs

```diff
@@ -776,6 +776,10 @@ fn altair_genesis(
 		},
 		ethereum: Default::default(),
 		evm: Default::default(),
+		liquidity_rewards_base: altair_runtime::LiquidityRewardsBaseConfig {
+			currency_id: CurrencyId::Native,
+			amount: altair_runtime::ExistentialDeposit::get(),
```

Comment:
It must be at least the `ExistentialDeposit` because else the `liquidity_rewards_base` sovereign account does not hold any balance on its own such that users cannot claim below the ED. We could also add this type to the `pallet_rewards::Config` and use it for the `GenesisBuild` but IMO the current approach is cleaner.

Adding this to the chain spec will only become effective if we need to update the Algol spec at some point upon restarting the network. Else we need to add the funds [via the pallets/rewards/src/migrations/new_instance.rs](https://github.com/centrifuge/centrifuge-chain/blob/main/pallets/rewards/src/migrations/new_instance.rs).

Quoting myself from https://github.com/centrifuge/centrifuge-chain/pull/1342

> While testing block rewards on a local chain environment, I stumbled into a minor issue which needs to be fixed: When claiming rewards, we [currently transfer](https://github.com/centrifuge/centrifuge-chain/blob/bd4a96bf6e9bb6606f6b285cfe0493b7415a8734/pallets/rewards/src/lib.rs#L370) from the pallet id of the  base `pallet_rewards` instance to the recipient with `keep_alive` enabled. This blocks any reward claims which would reduce the balance of the pallet's sovereign account below the existential deposit (ED). The `keep_alive` should stay enabled. Thus, we need to fund the `pallet_rewards` sovereign account (at least) with the ED which is done by the migration [added here](https://github.com/centrifuge/centrifuge-chain/pull/1342/files#diff-0946c08ffa10c8e8b0c1113bc058b68fda4c798a1724cad4c5b7e6241ce6cd6cR55).

Reply:
Anything that gets at least the `ED` in the native currency into the pallet's sovereign account does the trick.

Reply:
If you agree, I'll do a new PR with this in order to double-check with you if it would work leaving this as it is right now.

Reply:
Done: [38c8f51](https://github.com/centrifuge/centrifuge-chain/pull/1507/commits/38c8f5173d7f4ca5ba371fd966b6e17e6abd840c)